falcon.install

```diff
@@ -0,0 +1,47 @@
+<?php
+
+/**
+ * @file
+ * Install and uninstall functions for the Falcon installation profile.
+ */
+
+/**
+ * Implements hook_install().
+ *
+ * Performs actions to set up the site for this profile.
+ */
+function falcon_install() {
+  // Create default Commerce Store.
+  // TODO: consider moving it to install task (hook_install_tasks()) with
+  // configurable store values?
+  $address = [
+    'country_code' => 'US',
+    'address_line1' => '8 The Green',
+    'address_line2' => 'Ste A',
+    'locality' => 'Dover',
+    'administrative_area' => 'DE',
+    'postal_code' => '19901',
+  ];
+
+  /** @var \Drupal\commerce_store\StoreStorageInterface $commerce_store_storage */
+  $commerce_store_storage = \Drupal::entityTypeManager()->getStorage('commerce_store');
+  $store = $commerce_store_storage->create([
+    'type' => 'online',
+    'uid' => 1,
+    'name' => 'General Store',
+    'mail' => 'info@systemseed.com',
+    'address' => $address,
+    'default_currency' => 'EUR',
+    'billing_countries' => [],
+  ]);
+  $store->save();
+
+  $commerce_store_storage->markAsDefault($store);
+
+  // Create default consumer.
+  $consumer = \Drupal::entityTypeManager()->getStorage('consumer')->create([
```

Comment:
Can we move consumer creation to `falcon_common`? Not sure if this is the best place for it. Maybe, `falcon_common`? This is the only piece I can potentially accept in this file.